tox.ini

```diff
@@ -80,14 +80,18 @@ deps =
   -r {[vars]releases_test_path}/1.7/requirements.txt
 description = Test bundles
 
-[testenv:test_bundle_deployment]
+[testenv:test_bundle_deployment-{1.7,1.8}]
 commands =
     pytest -v --tb native --asyncio-mode=auto {[vars]test_path}/integration/test_bundle_deployment.py --keep-models --log-cli-level=INFO -s {posargs}
+setenv =
+    1.7: BUNDLE_PATH = "./releases/1.7/stable/kubeflow/bundle.yaml"
+    1.8: BUNDLE_PATH = "./releases/1.8/stable/kubeflow/bundle.yaml"
 deps = 
     aiohttp
     lightkube
     pytest-operator
     tenacity
     ops>=2.3.0
-    juju==2.9.43
+    1.7: juju<3.0.0
+    1.8: juju<4.0.0
 description = Test bundle deployment
```

Comment:
Both the BUNDLE_PATH and the juju versions seem buried in this tox file. I think we will need to change this as we release. Let's document this somewhere so people are aware and don't forget.